src/ControlSystem/Tags/FunctionsOfTimeInitialize.hpp

```diff
@@ -74,8 +74,23 @@ struct FunctionsOfTimeInitialize : domain::Tags::FunctionsOfTime,
           tmpl::list<>>,
       domain::OptionTags::DomainCreator<Metavariables::volume_dim>>;
 
+  /// @{
   /// This version of create_from_options is used if the metavariables did
   /// define control systems
+  template <
+      typename Metavariables, typename... OptionHolders,
+      Requires<tmpl2::flat_all_v<tt::is_a_v<Options::Auto, OptionHolders>...>> =
+          nullptr>
+  static type create_from_options(
+      const std::unique_ptr<::DomainCreator<Metavariables::volume_dim>>&
+          domain_creator,
+      const int measurements_per_update, const double initial_time,
+      const OptionHolders&... option_holders) {
+    return create_from_options<Metavariables>(
+        domain_creator, measurements_per_update, initial_time,
+        detail::get_option_holder(option_holders)...);
```

Comment:
Are these ever going to be anything except `Auto`s?  If not, just cast them to optionals, or add a method to `Auto` to do the cast so you don't have to deal with the template argument.